### ISSUE TITLE
carthage update fails for "2.6.0", is successful for "2.5.1".

### ISSUE THREAD

**Issue by losernobody** [skywinder/ActionSheetPicker-3.0#507]
- `carthage version`: 0.36.0
- `xcodebuild -version`: Xcode 11.7 Build version 11E801a

#### Command
```
carthage update --platform iOS --cache-builds
```

#### Cartfile
```
github "skywinder/ActionSheetPicker-3.0"
```

#### Output
```
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/username/Documents/appname/appname/Carthage/Checkouts/ActionSheetPicker-3.0/ActionSheetPicker-3.0.xcworkspace -scheme CoreActionSheetPicker -configuration Release -derivedDataPath /Users/username/Library/Caches/org.carthage.CarthageKit/DerivedData/11.7_11E801a/ActionSheetPicker-3.0/2.6.0 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/kj/p6q9x7557z7327jmrjpv1gv40000gp/T/ActionSheetPicker-3.0 SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/username/Documents/appname/appname/Carthage/Checkouts/ActionSheetPicker-3.0)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/kj/p6q9x7557z7327jmrjpv1gv40000gp/T/carthage-xcodebuild.dlNp7y.log
```

#### xcodebuild log
[carthage-xcodebuild.log](https://github.com/skywinder/ActionSheetPicker-3.0/files/5292427/carthage-xcodebuild.log)

#### PS
If Cartfile is changed to `github "skywinder/ActionSheetPicker-3.0" == 2.5.1`, carthage update is successful.

**Comment by skywinder** [comment]
oops, @noorulain17 probably related to last update

**Comment by noorulain17** [comment]
@skywinder not sure as we haven't changed anything related to Carthage.

**Comment by skywinder** [comment]
oh, it was SPM! my bad!

**Comment by ddaddy** [comment]
The problem is with `UIDatePickerStyleInline`. As you are using Xcode 11.7 this style is not defined.
I have issued a pull request https://github.com/skywinder/ActionSheetPicker-3.0/pull/508 that should fix it.

**Comment by ddaddy** [comment]
@losernobody can you maybe test it by changing your `Cartfile` to use `github "ddaddy/ActionSheetPicker-3.0" "develop"` before they merge it?

**Comment by noorulain17** [comment]
Fixed in #508
